src/core.ts

```diff
@@ -1075,7 +1075,7 @@ function applyHeadersMut(targetHeaders: Headers, newHeaders: Headers): void {
 }
 
 export function debug(action: string, ...args: any[]) {
-  if (typeof process !== 'undefined' && process.env['DEBUG'] === 'true') {
+  if (process?.env?.['DEBUG'] === 'true') {
```

Comment:
```suggestion
  if (typeof process !== 'undefined' && process?.env?.['DEBUG'] === 'true') {
```
we still need the typeof check for environments where `process` is not defined at all.